CONTRIBUTING.md

```diff
@@ -1,61 +1,92 @@
-<p align="center">
-  <picture>
-    <source srcset="./common/images/Kadena.JS_logo-white.png" media="(prefers-color-scheme: dark)"/>
-    <img src="./common/images/Kadena.JS_logo-black.png" width="200" alt="kadena.js logo" />
-  </picture>
-</p>
-
 # Contributing to Kadena.js
 
-:+1::tada: First off, thanks for taking the time to contribute! :tada::+1:
+## Table Of Contents
 
-The following is a set of guidelines for contributing to Kadena.js, which is hosted here 
-[Kadena.js](https://github.com/kadena-io/kadena.js) on GitHub.
+- Development
+- Tests
+- Making a Pull Request
+- Conventions
+- Publishing Packages
 
-### Table Of Contents
+Do you want to file a bug? Please [open a new issue][1].
 
-- Code of Conduct
-- Conventions
-- Issues
-- Pull Requests
+## Development
+
+Please [install Node.js][2] and [install pnpm][3] if you haven't already.
+Install Rush:
+
+```bash
+pnpm install --global @microsoft/rush
+```
+
+Clone the repository and install dependencies:
+
+```bash
+git clone git@github.com:kadena-community/kadena.js.git
+cd kadena.js
+rush install
+```
```

Comment:
Let's include `rush build` here as well. Some of the sub-projects will not run if `rush build` was not executed, and will throw arbitrary errors.

Reply:
That certainly doesn't hurt, I've added it.

Reply:
Should we add steps that are required when you switch branches?
- `rush update`: make sure to have the right dependencies
- `rush build -t <project you want to work on>` this creates all `d.ts` files of dependencies that are required for codehinting in IDEs

Should we add that running `rush update --full` has the effect of upgrading to the latest version that is allowed by the version indicators `(^|~|*)` and is probably not desirable or the intent of the person running it

Reply:
I've added more links to Rush tutorials, no need to duplicate such content and they probably explain it best :)